src/plugins/console/server/lib/proxy_request.ts

```diff
@@ -34,6 +34,16 @@ interface Args {
   rejectUnauthorized?: boolean;
 }
 
+/**
+ * Node http request library does not expect there to be trailing "[" or "]"
+ * characters in ipv6 host names but the Node URL class with helpers (from the "url" module)
+ * does not scrub these from the "hostname" value.
+ *
+ * We do this just before sending the request.
+ */
+const scrubURLHostname = (hostName: string): string =>
```

Comment:
nit: haven't seen the word "scrub" much around 😊 I would prefer `sanitizeHostname(...)` but that's just a preference.